docs/notebooks/demo.ipynb

```diff
@@ -191,6 +224,8 @@
    "source": [
     "import matplotlib.pyplot as plt\n",
     "\n",
+    "print(\"Known alphas dictionary: \", agent.arms[1].learner.known_alphas_)\n",
```

Comment:
Not sure this is adding much clarity, what do you think?

Reply:
Agreed, I took out the `known_alphas` reference in the "aside" part of the text too